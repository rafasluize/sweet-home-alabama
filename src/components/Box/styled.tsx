import styled from 'styled-components';

export const Box = styled.div`
  width: 400px;
  height: 400px;
  background: #ffffff;
  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.25);
  border-radius: 4px;
  padding: 24px;
`;

export default Box;
