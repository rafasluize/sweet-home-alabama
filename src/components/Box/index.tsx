import React, { FC } from 'react';
import * as Styled from './styled';

export const Box: FC = ({ children }) => <Styled.Box>{children}</Styled.Box>;

export default Box;
