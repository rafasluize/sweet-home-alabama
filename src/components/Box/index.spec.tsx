import { render } from '@testing-library/react';
import Box from './index';

describe('Box', () => {
  it('base styles', () => {
    const { container } = render(<Box />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
