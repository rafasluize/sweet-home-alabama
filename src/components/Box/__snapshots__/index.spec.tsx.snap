// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box base styles 1`] = `
<div
  class="sc-bczRLJ hfoEtf"
/>
`;
